### PR TITLE
WRP-9600: Update dependencies for console and copilot

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist 

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to their latest released version, and adapted the code accordingly.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated cpy-cli ^2.0.0  →   ^4.2.0, mapbox-gl ^0.54.1  →  ^1.13.3, and query-string ^6.14.1  → ^8.1.0 dependencies for console, copilot and components.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-9600

### Comments

Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: console/src/components/MapCore/MapCore.module.less [hunk 1]
```diff
@@ -74,4 +74,5 @@
 // we don't have control over and can't get back to our app from.
 :global(.mapboxgl-control-container) * {
 	pointer-events: none;
-}
+	display: none;
+}
```
Comment (daniel-stoian-lgp) [inline console/src/components/MapCore/MapCore.module.less]: please leave an empty line at the end of the file

File: console/package.json [hunk 2]
```diff
@@ -40,16 +40,16 @@
 		"classnames": "^2.3.2",
 		"ilib": "^14.15.1",
 		"immer": "^9.0.19",
-		"mapbox-gl": "^0.54.1",
+		"mapbox-gl": "^2.13.0",
```
Comment (daniel-stoian-lgp) [inline console/package.json]: did you check the release information for this dependency?
I see this breaking change: https://github.com/mapbox/mapbox-gl-js/releases?page=7

[v2.0.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0)
⚠️ Breaking changes
mapbox-gl-js is no longer under the 3-Clause BSD license. By upgrading to this release, you are agreeing to [Mapbox terms of service](https://www.mapbox.com/legal/tos/). Refer to LICENSE.txt for the new licensing terms and details. For questions, contact our team at https://support.mapbox.com/.



I think we should keep version 1.x as a dependecy to keep the open source use

File: console/src/components/MapCore/MapCore.module.less [hunk 3]
```diff
@@ -74,4 +74,5 @@
 // we don't have control over and can't get back to our app from.
 :global(.mapboxgl-control-container) * {
 	pointer-events: none;
-}
+	display: none;
```
Comment (daniel-stoian-lgp) [inline console/src/components/MapCore/MapCore.module.less]: check if this change is still needed after downgrading to version 1.x

**Review by daniel-stoian-lgp — COMMENTED** [review]
pelase check my comments

**Comment by daniel-stoian-lgp** [comment]
can you please update `query-string` dependency also in "components" ?

File: console/package.json (hunk 2 above)
Comment (andreitirlalge) [reply console/package.json]: I have decided to roll back to version ^1.13.0

File: console/package.json [hunk 4]
```diff
@@ -40,16 +40,16 @@
 		"classnames": "^2.3.2",
 		"ilib": "^14.15.1",
 		"immer": "^9.0.19",
-		"mapbox-gl": "^0.54.1",
+		"mapbox-gl": "^1.13.0",
```
Comment (daniel-stoian-lgp) [inline console/package.json]:
```suggestion
		"mapbox-gl": "^1.13.3",
```

File: console/src/components/MapCore/MapCore.module.less (hunk 3 above)
Comment (andreitirlalge) [reply console/src/components/MapCore/MapCore.module.less]: This change was not needed and removed after the rollback to version 1.13.0

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]

**Pull request merged**